diagnostics.sln

```diff
@@ -171,6 +171,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DotnetMonitor.UnitTests", "
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EventPipeTracee", "src\tests\EventPipeTracee\EventPipeTracee.csproj", "{870A7AD0-B808-491E-92F6-433BE990B374}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Diagnostics.DotNetMonitor", "src\Microsoft.Diagnostics.DotNetMonitor\Microsoft.Diagnostics.DotNetMonitor.csproj", "{D195D1B5-BC1F-4C24-A2C3-7B9900E1FB99}"
```

Comment:
Did you want to name this with some reference to the fact that it's really meant for our image scenario? DotNetMonitor.SelfContained.csproj or DotNetMonitor.Image.csproj?

Reply:
While that's the intended use, it's not what is really produced from this project. I think that projects should be named for what they produce, not for what their outputs are used.